tap_exchangeratesapi/__init__.py

```diff
@@ -55,9 +55,9 @@ def do_sync(base, start_date):
         while datetime.strptime(next_date, DATE_FORMAT) <= datetime.utcnow():
             logger.info('Replicating exchange rate data from %s using base %s',
                         next_date,
-                        base)
+                        "EUR")
```

Comment:
Is it because the free tier only supports "EUR"? I think it's better to support other currencies. If the API returning an error because the account is in free tier, we can return appropriate error message. What do yo think mas @rachmankamil?

Reply:
the response should be printed on stdout, that should be address the concern @farizap